CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+		
+		ruling_party_ideology = conservative
+		
+		has_country_modifier = dirty_war
+		
+		OR = { 
+		    has_country_flag = ITA_P2_Party_fail2 
+			AND = { has_country_flag = ITA_P2_Econ_fail has_country_flag = ITA_P2_Mafia_fail } 
+			AND = { has_country_flag = ITA_P2_Econ_fail has_country_flag = ITA_P2_Party_fail } 
+			AND = { has_country_flag = ITA_P2_Mafia_fail has_country_flag = ITA_P2_Party_fail } 
+		}
+		has_country_flag = ITA_P2
+		has_country_flag = ITA_P2_4
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { months = 3 }
+	
+	option = {
+		name = "Democratization is halted, Borghese assumes power"
+		badboy = 1
+		
+		ruling_party_ideology = nationalist
+		
+		#Leadership
+		remove_country_modifier = expert_diplomat
+		remove_country_modifier = successful_industrialist
+		remove_country_modifier = ivory_tower_intellectual
+		remove_country_modifier = man_of_the_people
+		remove_country_modifier = landed_gentleman
+		remove_country_modifier = military_man
+		remove_country_modifier = lord_admiral
+		remove_country_modifier = raving_loon
+		remove_country_modifier = luddite
+		remove_country_modifier = great_statesman
+		remove_country_modifier = backroom_boy
+		remove_country_modifier = great_reformer
+		remove_country_modifier = gauche_buffoon
+		remove_country_modifier = administrative_genius
+		remove_country_modifier = visionary_thinker
+		remove_country_modifier = competent_placeholder
+		remove_country_modifier = imperious_autocrat
+		remove_country_modifier = prince_of_terror
+		remove_country_modifier = efficient_sociopath
+		remove_country_modifier = pig_headed_isolationist
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = { name = lord_admiral duration = -1 }
+		
+		any_pop = {  
+		    limit = { is_primary_culture = yes }
+			ideology = { factor = 0.25 value = nationalist }
+			consciousness = -2
+			militancy = -2
+		}
+		
+		random_country = {
+			limit = { tag = USA OR = { government = democracy government = democracy1 government = presidential_dictatorship } is_greater_power = yes }
+			diplomatic_influence  = { who = THIS value = -20 }
+			relation = { who = THIS value = -25 }
+        }
+        
+		clr_country_flag = ITA_P2
+		clr_country_flag = ITA_P2_4
+        set_country_flag = ITA_National_Front_rule
+	}
+	
+}
+
+#Democratic Reorganization - The End
+country_event = {
+  	id = 8020423
+  	title = "The End of the Reorganization"
+  	desc = EVT_8020423_DESC
+  	picture = "ITA_1948_elections"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1973
+		NOT = { year = 1992 }
+		
+		government = populist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = conservative
+		
+		has_country_modifier = dirty_war
+		
+		has_country_flag = ITA_P2
+		has_country_flag = ITA_P2_4
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 6 }
+	
+	immediate = {
```

Comment:
Please put this in the event options instead as it will cause the text in the event to spill out of the UI box